src/app/cli/src/init/client.ml

```diff
@@ -548,21 +558,28 @@ let delegate_stake_graphql =
       ~doc:"PUBLICKEY Public key to which you want to delegate your stake"
       (required public_key_compressed)
   in
-  let genesis_constants = Genesis_constants.Compiled.genesis_constants in
-  let compile_config = Mina_compile_config.Compiled.t in
+  let config_file = Cli_lib.Flag.conf_file in
   let args =
-    Args.zip2
-      (Cli_lib.Flag.signed_command_common
-         ~minimum_user_command_fee:genesis_constants.minimum_user_command_fee
-         ~default_transaction_fee:compile_config.default_transaction_fee )
-      receiver_flag
+    Args.zip3 Cli_lib.Flag.signed_command_common receiver_flag config_file
   in
+
   Command.async ~summary:"Delegate your stake to another public key"
     (Cli_lib.Background_daemon.graphql_init args
        ~f:(fun
             graphql_endpoint
-            ({ Cli_lib.Flag.sender; fee; nonce; memo }, receiver)
+            ({ Cli_lib.Flag.sender; fee; nonce; memo }, receiver, config_file)
           ->
+         let open Deferred.Let_syntax in
+         let%bind compile_config =
+           let logger = Logger.create () in
+           let%map conf =
+             Runtime_config.Constants.load_constants ~logger config_file
+           in
+           Runtime_config.Constants.compile_config conf
+         in
+         let fee =
+           Option.value ~default:compile_config.default_transaction_fee fee
```

Comment:
This looks a bit odd to me: we're adding a config file parameter just to extract a single number.

I reviewed usages of `default_transaction_fee` in the codebase and it seems right to do the following:

1. Remove `default_transaction_fee` from node config, compile config and everywhere else
2. For CLI flag defaults (only place where the values is used) just specify manual default `0.25` (this value is in `devnet` and `mainnet`, and `dev` profile isn't really used for running CLI tools)

Reply:
Or we could place `0.25` to unconfigurable constants

Reply:
> IMO the default fee should be based on what's in the mempool, and neither the old solution, my solution, nor suggested solutions stand out as amazing.

This is definitely for future.

Neither solution stands out as amazing, but hardcoding `0.25` as the default seems still better. It feels awkward to have a config file parameter to take only default fee for a command that has a fee argument as well. And no other command uses this default transaction fee value from the config.

So, I'm removing default transaction fee from all configs and hardcode the existing value.